README.md

```diff
@@ -1,5 +1,36 @@
-# `aurora-cli-rs`
+<p>&nbsp;</p>
+<p align="center">
+<img src="https://raw.githubusercontent.com/hskang9/aurora-cli-rs/main/img/aurora-cli-logo.png" width=500>
+</p>
 
-A command-line interface for interacting with [Aurora](https://aurora.dev/).
+<p align="center">
+<strong>An instant, zero-config Aurora engine operator</strong>
+</p>
 
-See also prior art [aurora-cli](https://github.com/aurora-is-near/aurora-cli).
+<br/>
+
+## What is Engine?
+
+[Aurora](https://doc.aurora.dev/getting-started/aurora-engine/) is an Ethereum Virtual Machine (EVM) project built on the NEAR Protocol, that provides a solution for developers to deploy their apps on an Ethereum-compatible, high-throughput, scalable and future-safe platform, with low transaction costs for their users. Engine is the Aurora's implementation for it.
+
+## What is Aurora CLI?
+Aurora CLI is a command line interface to bootstrap Aurora engine with rapid speed built with rust.
+
+Aurora CLI comes preconfiguration with opinionated, sensible defaults for standard testing environments. If other projects mention testing on Aurora, they are referring to the settings defined in this repo.
+
+**Aurora CLI has the following advantages over api:**
+
+- :pencil: **Easily modifiable EVM states through terminal**
+- :handshake: **Quick to interact for rapid iterations**
+
+See also prior version [aurora-cli](https://github.com/aurora-is-near/aurora-cli).
+
+## Prerequisites
+- :crab: Rust
+
+## Quickstart
+
+- 📦 Install `aurora-cli-rs` and start interacting with it: 
+    *`cargo install aurora-cli-rs`* 
+- 🔍 Check out what each command is for in the official Aurora [docs](https://doc.aurora.dev/tools/aurora-cli)
```

Comment:
Noting dependency on https://github.com/aurora-is-near/doc.aurora.dev/pull/78/
These PRs should be merged together so that the link here is not broken.